static/js/containers/SearchPage_test.js

```diff
@@ -107,6 +107,19 @@ describe("SearchPage", () => {
       )
     })
   })
+  ;["", "a"].forEach(query => {
+    it("doesn't run a search if initial search text is empty or < min characters", async () => {
```

Comment:
Can we alter the test name based on what `query` is?

Reply:
✔️